content/pages/meetup/2024/1023-nycu.rst

```diff
@@ -0,0 +1,147 @@
+========================================
+Meetup 2024 October 23th in NYCU
+========================================
+
+:date: 2024-10-18 17:25
+:url: meetup/2024/1023-nycu
+:save_as: meetup/2024/1023-nycu.html
+
+Date
+-----
+
+* Date: October 16th, 2024
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+There will be many different subjects discussed at the same time.
+
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
```

Comment:
The table style may use some improvement (now or in the future):

<img width="1091" alt="image" src="https://github.com/user-attachments/assets/521d75aa-e00a-44ad-a1bf-f5c0cf279f31">

It does not look very clear when the items happen.